src/main/java/com/kodilla/ecommercee/domain/Order.java

```diff
@@ -1,4 +1,43 @@
 package com.kodilla.ecommercee.domain;
 
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.experimental.FieldDefaults;
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.time.LocalDateTime;
+
+@Getter
+@AllArgsConstructor
+@NoArgsConstructor
+@FieldDefaults(level = AccessLevel.PRIVATE)
```

Comment:
jeżeli używasz `@FieldDefaults(level = AccessLevel.PRIVATE)`, to `private` bezpośrednio na polach jest już niepotrzebne